parser-typechecker/src/Unison/Typechecker/Context.hs

```diff
@@ -997,7 +999,9 @@ wantRequest ::
   Term v loc ->
   Type v loc ->
   (Type v loc, Wanted v loc)
-wantRequest loc ~(Type.Effect'' es t) = (t, fmap (Just loc,) es)
```

Comment:
GHC got confused and even though this pattern is irrefutable  it wouldn't believe that the patterns were exhaustive, even if I added the `COMPLETE` pragma for `Effect''`, so I just used `unEffect0` instead, which is all that the `Effect''` pattern does.